en/products/relay/matrix.ftl

```diff
@@ -0,0 +1,70 @@
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/relay/#pricing
+
+plan-matrix-title = Pricing
+
+## PRICING MATRIX SECTION
+
+plan-matrix-offer-title = Choose a level of protection that’s right for you
+plan-matrix-offer-body = { -brand-name-firefox-relay } protects your identity and your inbox with unique email masks. Add on even more security with { -brand-name-relay } phone masking, and save { $savings } by adding { -brand-name-vpn } protection.
+
+## Top Row Labels
+
+# Variables:
+#   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
+plan-matrix-heading-features = Features
+plan-matrix-heading-plan-free = Limited email protection
+plan-matrix-heading-plan-premium = Email protection
+plan-matrix-heading-plan-phones = Email & phone protection
+plan-matrix-heading-plan-bundle-2 = Add { -brand-name-vpn } protection
+
+## Feature Breakdowns
+
+plan-matrix-feature-email-masks = Email masks to protect your real email address
+plan-matrix-feature-browser-extension = Browser extension to use { -brand-name-relay } on any site
+plan-matrix-feature-email-tracker-removal = Remove email trackers
+plan-matrix-feature-promo-email-blocking = Block promotional emails
+plan-matrix-feature-email-subdomain = { -brand-name-relay } email domain to create masks on-the-go
+plan-matrix-feature-email-reply = Reply to emails anonymously
+plan-matrix-feature-phone-mask = Phone mask to protect your real phone number
+plan-matrix-feature-mozvpn = { -brand-name-mozilla-vpn }
+
+## Feature Breakdowns Mobile (Shorter than desktop strings)
+
+plan-matrix-feature-mobile-email-masks = Email masks
+plan-matrix-feature-mobile-browser-extension = Browser extension
+plan-matrix-feature-mobile-email-tracker-removal = Remove email trackers
+plan-matrix-feature-mobile-promo-email-blocking = Block promotional emails
+plan-matrix-feature-mobile-email-subdomain = Unique { -brand-name-relay } email domain
+plan-matrix-feature-mobile-email-reply = Reply to emails anonymously
+plan-matrix-feature-mobile-phone-mask = Protect your real phone number
+plan-matrix-feature-mobile-mozvpn = { -brand-name-mozilla-vpn }
+
+## Plan Details
+
+plan-matrix-heading-price = Price
+plan-matrix-feature-count-unlimited = Unlimited
+plan-matrix-feature-included = Included
+plan-matrix-feature-not-included = Not included
+plan-matrix-price-free = Free
+
+## Plan Price
+
+# Variables:
+#   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
+plan-matrix-price-monthly-calculated = { $monthly_price }/mo.
+plan-matrix-price-period-yearly = Yearly
+plan-matrix-price-period-monthly = Monthly
+plan-matrix-price-period-yearly-footnote-1 = Billed yearly
+plan-matrix-price-period-monthly-footnote-1 = Billed monthly
+plan-matrix-price-vpn-discount-promo = <span>Save { $savings }</span> on regular { -brand-name-vpn } price
```

Comment:
```suggestion
plan-matrix-price-vpn-discount-promo = <span>Save { $savings }</span> on regular VPN price
```

Reply:
What about this instead?

`plan-matrix-price-vpn-discount-promo = <span>Save { $savings }</span> on regular { -brand-name-mozilla-vpn } price`

Reply:
Using the full brand term would work here. (Note, I think we changed this string once already to include a %. Pointing it out so that change doesn't get reverted.)

Reply:
Decided against using the brand term. Updating to use the recommended string.